### PR TITLE
Various ideas

### PR DESCRIPTION
**Pull request by chaosvolt**
So, this is a bunch of ideas I figured I'd self-PR in-between my fiddling with ideas for an attempt at natural magic in CDDA.

* Reduced energy drain rate of the mantle of shadows by half.
* Changed weather effect of symbol of judgement to be on-wield, gave it an effect on machines.
* Changed the hellfire stave's forced burst to instead be optional.
* Changed shot effect on wraithslayer to use "wide" instead of "shot".
* Quality of life change: most professions (except apprentice) get at least some access to a knife, firestarter, and container. Trivial to improvise most of those innawoods, but it's largely just a brief bit of tedium.
* Gave thunder gauntlets the option to use the shockstorm artifact effect in addition to tazing things.
* Small description change for symbol of judgement, sooomeone thought from the description that the 5-volume lightning weapon was wearable.

One thing I wanted to do was rebalance magic ranged weapons, but the `FIRE_X` flag isn't actually modular. Only allows 20, 50, and 100 it seems. None of which really work for the amounts I had in mind. :V

### CODE REVIEW

**Comment by DangerNoodle** [comment]
No sign of change to artifact effect of mask of insight.

**Comment by chaosvolt** [comment]
Doh. Then again, if https://github.com/CleverRaven/Cataclysm-DDA/pull/21721 gets merged I can leave it at the basic clairvoyance because 8 sounds like a better range.

File: Arcana/professions.json [hunk 1]
```diff
@@ -153,11 +155,13 @@
...
             "male": [
-                "boxer_briefs"
+                "boxer_briefs",
+                "pants"
             ],
             "female": [
                 "bra",
-                "panties"
+                "panties",
+                "dress_shirt"
```
Comment (BorkBorkGoesTheCode) [inline Arcana/professions.json]: Two dress shirts?

Comment (chaosvolt) [reply Arcana/professions.json]: ...doh. Was supposed to be a skirt.

**Pull request merged**